### PR TITLE
Create verified-builds.md

### PR DESCRIPTION
**Pull request by Woody4618**
### Problem
The Verify program project will soon move into the anzo tool suite. We don't have official docs on it yet. 

### Summary of Changes
Added docs for verify program and Security.txt

### CODE REVIEW

File: content/guides/advanced/verified-builds.md
Comment (nickfrosty) [inline content/guides/advanced/verified-builds.md:341]:
```suggestion
  `query-security-txt` tool, which can validate both onchain programs and local
```

File: content/guides/advanced/verified-builds.md
Comment (nickfrosty) [inline content/guides/advanced/verified-builds.md:3]: lets make this featured to show at the top of the guides page
```suggestion
difficulty: intermediate
featured: true
```

File: content/guides/advanced/verified-builds.md
Comment (Woody4618) [reply content/guides/advanced/verified-builds.md:61]: It was supposed to shoe all the fields of the PDA. In included them in the text instead.

File: content/guides/advanced/verified-builds.md
Comment (Woody4618) [reply content/guides/advanced/verified-builds.md:150]: It needs to have a cargo.lock. But for that you need the cargo toml and then create the lock file. Updated.

File: content/guides/advanced/verified-builds.md
Comment (Woody4618) [reply content/guides/advanced/verified-builds.md:3]: I would actually like to not put too much featuring on it until its actually ready. Wdyt? Still lots a work to do until its properly usable.

Comment (nickfrosty) [reply content/guides/advanced/verified-builds.md:3]: that's fair then, we can ignore making it featured.

do you think we should generally wait until we merge this then?

Comment (jacobcreech) [reply content/guides/advanced/verified-builds.md:3]: ship imo. Feature later. Get it indexed

File: content/guides/advanced/verified-builds.md
Comment (jacobcreech) [inline content/guides/advanced/verified-builds.md:4]:
```suggestion
title: "How to Verify a Program"
```

File: content/guides/advanced/verified-builds.md
Comment (jacobcreech) [inline content/guides/advanced/verified-builds.md:38]:
```suggestion
> A verified build should not be considered more secure than an unverified build.
> A verified gives the user the ability to verify that the program was built
```

File: content/guides/advanced/verified-builds.md
Comment (jacobcreech) [inline content/guides/advanced/verified-builds.md:41]:
```suggestion
> While a verified build should not be considered more secure than an unverified build, the build enables developers to self verify the source code matches what is deployed onchain. Using the source code, a developer can then validate what the code executes when sending a transaction.
```

File: content/guides/advanced/verified-builds.md
Comment (jacobcreech) [inline content/guides/advanced/verified-builds.md:92]:
```suggestion
# How do I create verified builds?
```

**Review by jacobcreech — APPROVED** [review]

**Pull request merged**
